src/engine/games.js

```diff
@@ -1,46 +1,61 @@
-const Game = require('./game');
-const { NORMAL, PRACTICE } = require('./game-types');
-
-const GAME_REMOVE_TIME = 4 * 60 * 1000;
+const OpenGame = require("./games/open-game");
+const PracticeGame = require("./games/practice-game");
 
 class Games {
-  constructor(highScoreRepository) {
+  constructor(config, highScoresRepository) {
     this.openGames = {};
     this.runningGames = {};
     this.practiceGames = {};
     this.spectators = [];
-    this.highscores = highScoreRepository;
+    this.highScoresRepository = highScoresRepository;
+    this.config = config;
   }
 
-  newGame() {
-    const game = new Game(NORMAL);
+  newOpenGame() {
+    const game = new OpenGame(
+      {
+        MINIMUM_PLAYERS: this.config.MINIMUM_PLAYERS,
+        MAXIMUM_PLAYERS: this.config.MAXIMUM_PLAYERS,
+        LOCK_COUNTDOWN: this.config.LOCK_COUNTDOWN,
+        PRE_LOCK_COUNTDOWN: this.config.PRE_LOCK_COUNTDOWN,
+        GAME_TICK: this.config.GAME_TICK,
+        MAX_GAME_TICKS: this.config.MAX_GAME_TICKS
+      },
+      this.highScoresRepository
+    );
 
     this.openGames[game.getId()] = game;
 
     return game;
   }
 
-  newPractice() {
-    const game = new Game(PRACTICE);
+  newPracticeGame() {
```

Comment:
Might be a good update if gamesConfig grows to introduce getGameConfig method:

`new PracticeGame({
...this.getGameConfig(this.config),
MINIMUM_PLAYERS: 1,
MAXIMUM_PLAYERS: 1,
})`